src/store.js

```diff
@@ -4,9 +4,10 @@ import register from "./register/store/Register";
 import profile from "@/profile/store";
 import * as signin from "@/signin/store/signin.js";
 import * as signout from "@/signout/store/signout.js";
+import * as companies from "@/companies/store/companies.js";
+import * as employees from "@/employees/store/employees.js";
```

Comment:
Rename file the `store/employees.js` to `store/index.js`
Then we can import as below 
```suggestion
import * as employees from "@/employees/store";
```